src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int MAX_SIZE = 25;
+    private static final int DEFAULT_RADIUS = 10;
+    private static final int RANDOM_FIGURE_NUMBER = 4;
+    private static final String DEFAULT_COLOR = Color.WHITE.toString();
+
+    private Random random = new Random();
+
+    public int getRandom() {
+        return random.nextInt(MAX_SIZE);
+    }
+
+    public void setRandom(Random random) {
+        this.random = random;
+    }
+
+    public Figure getDefaultFigure() {
+        Figure circle = new Circle(DEFAULT_RADIUS);
+        circle.setColor(DEFAULT_COLOR);
+        System.out.println("Figure: circle, area: " + circle.getArea()
+                + " sq.units, radius: " + DEFAULT_RADIUS + " units, color: "
+                + DEFAULT_COLOR.toLowerCase());
+        return circle;
+    }
+
+    public Figure getRandomFigure() {
+        int index = random.nextInt(RANDOM_FIGURE_NUMBER);
+        switch (index) {
+            case 0:
+                return getRandomCircle();
+            case 1:
+                return getRandomSquare();
+            case 2:
+                return getRandomRectangle();
+            case 3:
+                return getRandomIsoscelesTrapezoid();
+            case 4:
+                return getRandomRightTriangle();
+            default:
+                break;
+        }
+        return new Figure();

```

Comment:
```suggestion
            default:
               return getDefaultFigure();
        }

```

I think it is better than return empty figure